src/cubic_loader/qlik/ods_tables.py

```diff
@@ -12,4 +13,15 @@
     "EDW.CARD_DIMENSION",
     "EDW.DEVICE_DIMENSION",
     "EDW.USE_TRANSACTION",
+    # comp b
+    "EDW.PAYMENT_SUMMARY",
```

Comment:
I ran this locally and all of the tables loaded successfully except `EDW.PAYMENT_SUMMARY`.

The `dfm` file for table snapshot starts with these columns:
```
{
	"ordinal":	1,
	"name":	"header__change_seq",
	"type":	"STRING",
	"length":	35,
	"precision":	0,
	"scale":	0,
	"primaryKeyPos":	0
}, {
	"ordinal":	2,
	"name":	"header__change_oper",
	"type":	"STRING",
	"length":	1,
	"precision":	0,
	"scale":	0,
	"primaryKeyPos":	0
}, {
	"ordinal":	3,
	"name":	"header__timestamp",
	"type":	"DATETIME",
	"length":	0,
	"precision":	0,
	"scale":	0,
	"primaryKeyPos":	0
}, {
	"ordinal":	4,
	"name":	"SETTLEMENT_DAY_KEY",
	"type":	"INT4",
	"length":	10,
	"precision":	0,
	"scale":	0,
	"primaryKeyPos":	1
}, {
	"ordinal":	5,
	"name":	"TRANSIT_DAY_KEY",
	"type":	"INT4",
	"length":	10,
	"precision":	0,
	"scale":	0,
	"primaryKeyPos":	2
}, {
	"ordinal":	6,
	"name":	"POSTING_DAY_KEY",
	"type":	"INT4",
	"length":	10,
	"precision":	0,
	"scale":	0,
	"primaryKeyPos":	3
}, {
	"ordinal":	7,
	"name":	"OPERATING_DAY_KEY",
	"type":	"INT4",
	"length":	10,
	"precision":	0,
	"scale":	0,
	"primaryKeyPos":	4
}, {
	"ordinal":	8,
	"name":	"TXN_SOURCE",
	"type":	"STRING",
	"length":	30,
	"precision":	0,
	"scale":	0,
	"primaryKeyPos":	5
}, {
	"ordinal":	9,
	"name":	"PAYMENT_TYPE_KEY",
	"type":	"INT2",
	"length":	5,
	"precision":	0,
	"scale":	0,
	"primaryKeyPos":	6
}, {
	"ordinal":	10,
	"name":	"SALES_CHANNEL_KEY",
	"type":	"INT2",
	"length":	5,
	"precision":	0,
	"scale":	0,
	"primaryKeyPos":	7
}, {
	"ordinal":	11,
	"name":	"OPERATOR_KEY",
	"type":	"INT4",
	"length":	10,
	"precision":	0,
	"scale":	0,
	"primaryKeyPos":	8
}, {
	"ordinal":	12,
	"name":	"DEVICE_KEY",
	"type":	"INT4",
	"length":	10,
	"precision":	0,
	"scale":	0,
	"primaryKeyPos":	9
}, {
	"ordinal":	13,
	"name":	"FARE_PROD_KEY",
	"type":	"INT4",
	"length":	10,
	"precision":	0,
	"scale":	0,
	"primaryKeyPos":	10
}, {
	"ordinal":	14,
	"name":	"REASON_KEY",
	"type":	"NUMERIC",
	"length":	0,
	"precision":	10,
	"scale":	0,
	"primaryKeyPos":	11
}
```

This shows 11 primary key columns, however when creating a table, including these columns as part of the primary key, the initial snapshot load fails because several of the primary key columns contain NULL values. 

It is a requirement for Postgres that any column included in the primary key also be NON-NULL. 

Cubic must be using a database that allows NULL values in the primary key???